django_netjsongraph/static/netjsongraph/js/visualize.js

```diff
@@ -63,4 +78,39 @@ django.jQuery(function($) {
             e.preventDefault();
         }
     }
+
+    var getTopologyHistory = function(e) {
+        query_date = $('#dp').val();
+        date = query_date.split('/').reverse();
+        if(date!==[]){
+            var x = date[1];
+            date[1] = date[2];
+            date[2] = x;
+        }
+        date = date.join('-');
+        graph_url = $('.switcher').attr('graph-url')+'?date='+date;
+        body = $('body');
+        $.get(graph_url, function(html) {
+            inner.html(html);
+            $('svg').appendTo(inner);
+            $('.njg-overlay').appendTo(inner);
+            $('.njg-metadata').appendTo(inner);
+            overlay.find('#dp').val(query_date);
+            overlay.show();
+            body.css('overflow', 'hidden');
+            inner.css('overflow', 'hidden');
+            overlay.find('.close').click(function(e){
+                e.preventDefault();
+                closeOverlay();
+            })
+            overlay.find('#submit').click(function(e){
+                e.preventDefault();
+                getTopologyHistory();
+            })
+        }).
+        error(function(xhr) {
+            if (xhr.status == 400) {
+            }
+        });
```

Comment:
what about this section? can this situation be triggered by the user? If yes, add an alert with some text message, otherwise remove it